src/permissions/NativeTokenRollingSpendLimit/NativeTokenRollingSpendLimitPermission.sol

```diff
@@ -66,18 +74,27 @@ contract NativeTokenRollingSpendLimitPermission is IPermissionContract {
         ICoinbaseSmartWallet.Call[] memory calls = abi.decode(userOp.callData[4:], (ICoinbaseSmartWallet.Call[]));
         uint256 callsLen = calls.length;
         uint256 spendValue = 0;
-        // if no paymaster, set initial spendValue as requiredPrefund
-        /// @dev no accounting done for the refund step so ETH spend is overestimated slightly
-        if (userOp.paymasterAndData.length == 0) {
+
+        // increment spendValue if gas cost beared by the user
+        /// @dev No accounting done for the refund step so native token spend is overestimated slightly
```

Comment:
Style question/nit: wonder if we should leave `///` for true natspec at top of function, and just use `//` with no `@dev` in the function?